config_cfg.js

```diff
@@ -1,10 +1,16 @@
 /* eslint-disable no-unused-vars, no-var */
+
```

Comment:
Here is the same indentation like in previous pr... I'm a bit confused.

Reply:
Yeah, these are the same changes due to outdated file. I am surprised how you don't get these errors before merging this file to master?